.circleci/config.yaml

```diff
@@ -0,0 +1,120 @@
+version: 2.1
+commands:
+  install-cloudfriend:
+    description: "Install mapbox/cloudfriend"
+    steps:
+      - run: npm install -g @mapbox/cfn-config
```

Comment:
Looks like the npm install below (https://github.com/hotosm/tasking-manager/pull/1469/files#diff-19f445bfad95516224bbe1795cd87b36R34) will not install `@mapbox/cloudfriend`, since we have more than one `package.json` (One in the root directory and one in the `client/` directory). Any thoughts around how we can make this less messy? 😬 Otherwise, for now, we'll have to explicitly run npm install in the root TM directory as well.

Reply:
It's not quite less messy but the changes I just pushed I think are a little better way of doing things.